app/themes/map/ui/app/root/root.html

```diff
@@ -1,68 +1,29 @@
+    <ui-gmap-markers models="ctrl.markers" coords="'self'" icon="'icon'" fit="false" click="ctrl.markerClick">
+    </ui-gmap-markers>
+    <ui-gmap-window
+            coords='ctrl.infoWindow.coords'
+            show='ctrl.infoWindow.shown'
+            templateUrl='ctrl.infoWindow.templateUrl'
+            templateParameter='ctrl.infoWindow.data'
+            options='ctrl.infoWindow.options'
+            closeClick='ctrl.closeClick'
+            >
+    </ui-gmap-window>
+  </ui-gmap-google-map>
+
   </div>
-</div>
 
-<footer id="footer" class="powered">
-  <div class="container">
-    <div class="row">
-      <div class="col-sm-12">
-        <div class="social-links">
-          <a href="https://www.facebook.com/marklogic" title="Facebook" rel="external">
-            <img src="images/facebook.svg" alt="Facebook" title="Facebook"></a>
-          <a href="https://twitter.com/marklogic" title="Twitter" rel="external">
-            <img src="images/twitter.svg" alt="Twitter" title="Twitter"></a>
-          <a href="http://www.linkedin.com/company/marklogic" title="LinkedIn" rel="external">
-            <img src="images/linkedin.svg" alt="LinkedIn" title="LinkedIn"></a>
-          <a href="https://www.flickr.com/photos/marklogicuc/" title="Flickr" rel="external">
-            <img src="images/flickr.svg" alt="Flickr" title="Flickr"></a>
-          <a href="https://www.youtube.com/channel/UCT2Q5qhC2gmd89K9qpV2TrQ" title="YouTube" rel="external">
-            <img src="images/youtube.svg" alt="YouTube" title="YouTube"></a>
-          <a href="https://plus.google.com/+MarklogicCo/posts" title="Google+" rel="external">
-            <img src="images/google.svg" alt="Google+" title="Google+"></a>
-        </div>
-      </div>
-    </div>
-    <div class="row">
-      <div class="col-sm-12">
-        <ul id="menu-footer-navigation-logged-in" class="nav">
-          <li id="menu-item-140" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-140">
-            <a target="_blank" href="http://www.marklogic.com">MarkLogic.com</a>
-          </li>
-          <li id="menu-item-71" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-71">
-            <a target="_blank" href="http://developer.marklogic.com">MarkLogic Developers</a>
-          </li>
-        </ul>
-      </div>
-    </div>
-    <div class="row copyright">
-      <div class="col-sm-12 text-muted">
-        Copyright <sup>&copy;</sup> {{ ctrl.currentYear }} MarkLogic Corporation. MARKLOGIC <sup>&reg;</sup> is a registered trademark of MarkLogic Corporation.
-      </div>
-    </div>
+  <div id="hamburger" ng-click="ctrl.toggleControls()">
+    <span class="glyphicon glyphicon-menu-hamburger"></span>
   </div>
-</footer>
+  <div id="logo"></div>
+
+  <div id="map-controls"><ui-view /></div>
+
+  <div id="powered-by"></div>
```

Comment:
I think we need to preserve the footer somehow. The copyright notice should be visible..